skiko/src/jvmMain/kotlin/org/jetbrains/skiko/SkiaLayer.jvm.kt

```diff
@@ -98,10 +98,13 @@ actual open class SkiaLayer internal constructor(
         var isFullscreenDispatched: Boolean = false
         override fun componentShown(e: ComponentEvent) {
             if (isFullscreenDispatched) {
```

Comment:
We probably should check `_isFullscreen` here (and remove `isFullscreenDispatched`). Because:

1. This code won't preserve fullscreen:
```
layer.isFullscreen = true
window.isVisible = true
window.isVisible = false
window.isVisible = true
```

2. When the user manually exits fullscreen, `isFullscreenDispatched` will stay `true`.

Reply:
> We probably should check _isFullscreen

Not with if, but just:
```
backedLayer.isFullscreen = _isFullscreen
```

or
```
if (backedLayer.isFullscreen != _isFullscreen) {
   backedLayer.isFullscreen = _isFullscreen
}
```